app/src/main/java/org/techtown/assignment/ProfileFragment.kt

```diff
@@ -0,0 +1,47 @@
+package org.techtown.assignment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.fragment_profile.*
+import org.techtown.assignment.databinding.FragmentProfileBinding
+
+class ProfileFragment : Fragment() {
+    private var _binding: FragmentProfileBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentProfileBinding.inflate(layoutInflater)
+        initTransactionEvent()
+        return binding.root
+
+    }
+
+    private fun initTransactionEvent() {
+        val fragment1 = FollowerFragment()
+        val fragment2 = RepositoryFragment()
+
+        childFragmentManager.beginTransaction().add(R.id.fcv_profile, fragment1).commit()
+
+        binding.btnFollower.setOnClickListener {
+            btn_follower?.isSelected = btn_follower?.isSelected != true
```

Comment:
```suggestion
            btn_follower?.isSelected = true
```
팔로워 버튼을 누르면 isSelected를 그냥 true로 해주면 되지 않을까요? 고민을 꽤 깊게 하셨네요 :-)

Reply:
> 팔로워 버튼을 누르면 isSelected를 그냥 true로 해주면 되지 않을까요? 고민을 꽤 깊게 하셨네요 :-)

네... 해결방안 알고 나서 살짝 허무했씁니다 ㅠㅠ